recipes/gau2grid/build.sh

```diff
@@ -0,0 +1,30 @@
+if [ "$(uname)" == "Darwin" ]; then
+    ALLOPTS="${CFLAGS}"
+fi
+if [ "$(uname)" == "Linux" ]; then
+    # revisit when c-f moves to gcc8
+    # * checked Dec 2020 at gcc9 and define still needed
+    ALLOPTS="${CFLAGS} -D__GG_NO_PRAGMA"
+fi
+
+${BUILD_PREFIX}/bin/cmake \
+    -H${SRC_DIR} \
+    -Bbuild \
+    -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_C_COMPILER=${CC} \
+    -DCMAKE_C_FLAGS="${ALLOPTS}" \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    -DPYMOD_INSTALL_LIBDIR="/python${PY_VER}/site-packages" \
```

Comment:
This looks a little odd, I would expect this to be like below but maybe the CMake configuration doesn't handle absolute paths correctly and forces it to be the way you already did it?
```suggestion
    -DPYMOD_INSTALL_LIBDIR="$SP_DIR" \
```

Reply:
same as above